src/components/BlockingViews/FullPageNotFoundView.js

```diff
@@ -47,23 +41,16 @@ const defaultProps = {
     titleKey: 'notFound.notHere',
     subtitleKey: 'notFound.pageNotFound',
     linkKey: 'notFound.goBackHome',
-    shouldShowBackButton: true,
     shouldShowBackHomeLink: false,
-    shouldShowCloseButton: true,
-    onBackButtonPress: () => Navigation.dismissModal(),
+    onBackButtonPress: Navigation.goBack,
 };
 
 // eslint-disable-next-line rulesdir/no-negated-variables
 const FullPageNotFoundView = (props) => {
     if (props.shouldShow) {
         return (
             <>
-                <HeaderWithCloseButton
-                    shouldShowBackButton={props.shouldShowBackButton}
```

Comment:
Removal of `shouldShowBackButton` in this PR caused [this issue](https://github.com/Expensify/App/issues/21596).

Issue: Back Button is shown on desktop in Report Screen